includes/widgets/button.php

```diff
@@ -1,6 +1,8 @@
 <?php
 namespace Elementor;
 
+use Elementor\Modules\DynamicTags\Module;
```

Comment:
It's most recommended to use an alias for this `Module` class, since it's used outside of its context.

Maybe something like `DynamicTagsModule`.

Reply:
+1